KarhooUISDK/Screens/BookingScreen/QuoteListMVP/KarhooQuoteListPresenter.swift

```diff
@@ -174,14 +172,8 @@ extension KarhooQuoteListPresenter: BookingDetailsObserver {
             switch result {
             case .success(let quotes):
                 self?.quoteSearchSuccessResult(quotes, bookingDetails: details)
-                if details.destinationLocationDetails != nil,
-                    let scheduled = details.scheduledDate,
-                    let originTimeZone = details.originLocationDetails?.timezone() {
+                if details.destinationLocationDetails != nil {
```

Comment:
This is where you lost the segmented control. Just remove the code related to the quotes title inside the if. The if condition should look like `if details.destinationLocationDetails != nil, details.scheduledDate != nil { //hide the quote sorter }` The quote sorter should be hidden in prebook mode, but visible in asap mode